### DISCUSSION TITLE
Thinner line options for handwriting

### DISCUSSION THREAD

**Discussion by psuto** [zsviczian/obsidian-excalidraw-plugin#271] [Ideas]
Thinner line options for handwriting.

**Comment by zsviczian** [comment]
This might be a solution for you.
Help: https://zsviczian.github.io/obsidian-excalidraw-plugin/ExcalidrawScriptsEngine.html
YouTube demo: https://youtu.be/hePJcObHIso
The script you need: https://github.com/zsviczian/obsidian-excalidraw-plugin/blob/master/ea-scripts/Modify%20stroke%20width%20of%20selected%20elements.md
